test/post-911-gib-status/components/UserInfoSection.unit.spec.jsx

```diff
@@ -7,7 +7,30 @@ import UserInfoSection from '../../../src/js/post-911-gib-status/components/User
 
 const props = {
   enrollmentData: {
-    vaFileNumber: '12345678'
+    veteranIsEligible: true,
```

Comment:
I think we need a test for how the value of `activeDuty` affects the benefit end date warning.